apps/heureka/src/components/Services/ServiceDetails/ServiceDetails.tsx

```diff
@@ -3,21 +3,35 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from "react"
+import React, { useState, useEffect } from "react"
 import { ContentHeading, Stack, Badge, Pill, Label } from "@cloudoperators/juno-ui-components"
-import { ServiceImageVersions } from "../common/ServiceImageVersions"
+import { ServiceImageVersions, ServiceImageVersion } from "../common/ServiceImageVersions"
+import { ImageVersionDetailsPanel } from "../common/ImageVersionDetailsPanel/ImageVersionDetailsPanel"
 import { MessagesProvider, Messages } from "@cloudoperators/juno-messages-provider"
 import { useStore } from "../../../store/StoreProvider"
 import { SelectServiceDetailsPayload, UserView } from "../../../store/StoreProvider/types"
 import { Breadcrumb } from "../../common/Breadcrumb"
 
 export const ServiceDetails = () => {
   const { selectedView } = useStore()
+  const [selectedImageVersion, setSelectedImageVersion] = useState<ServiceImageVersion | null>(null)
+
   const selectedService =
     selectedView.viewId === UserView.ServiceDetails
       ? (selectedView as SelectServiceDetailsPayload).params.service
       : undefined
 
+  const selectedImageVersionFromStore =
```

Comment:
I think this logic can be set as a default param into the local state and remove the useEffect or?